src/middlewared/middlewared/plugins/failover_/datastore.py

```diff
@@ -37,17 +37,14 @@ def set_failure(self):
         try:
             self.send()
         except Exception:
-            self.logger.warning(
```

Comment:
Why remove this logging message? (But I think we should remove the traceback though)

Reply:
Agree, I added it back (without traceback)